runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -97,6 +97,7 @@ J9::Z::CodeGenerator::initialize()
        !TR::Compiler->om.canGenerateArraylets())
```

Comment:
Should we rename the JIT option `TR_DisableSIMDStringHashCode` as it is now used for vectorizedHashCode as well. I think it makes sense to rename to `TR_DisableSIMDHashCode`. 
And in place where we are recognizing the method in code-gen, we can use the environment flag to disable the String or Array hashcode acceleration (Before suppressing the inlining through traditional inliner).

Reply:
Should I also rename the code generator flag `SupportsInlineStringHashCode` to `SupportsInlineHashCode` and use that instead of adding a `SupportsInlineVectorizedHashCode` flag?

The `TR_DisableSIMDStringHashCode` option is defined in OMR, it might make more sense to migrate it to just be in OpenJ9, and rename it as part of that change.